app/models/csv_files/csv_file.rb

```diff
@@ -0,0 +1,147 @@
+class CsvFile < ActiveRecord::Base
+  attr_accessor :upload
+
+  STI = %W(WeamsCsvFile)
```

Comment:
Sounds good! Thanks for the explanation. Could you leave that in comment next to the one that is sticking around?